### PR TITLE
[Pages] Fix angular CLI docs

### LINKED ISSUE #12183: Build command Typo on https://developers.cloudflare.com/pages/framework-guides/deploy-an-angular-site/
### Which Cloudflare product does this pertain to?

Pages

### Existing documentation URL(s)

https://developers.cloudflare.com/pages/framework-guides/deploy-an-angular-site/

### What changes are you suggesting?

`npx ng build --configuration production`
**instead of** 
`npx ng build --prod`

### Additional information

The commond provided in the documentation is for older versions of Angular, if you use the new tool https://www.npmjs.com/package/create-cloudflare then the Build command needs to be updated to `npx ng build --configuration production`

### PR DESCRIPTION
**Pull request by markdembo**
Resolves https://github.com/cloudflare/cloudflare-docs/issues/12183

### CODE REVIEW

File: content/pages/_partials/_build-configuration.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ build_configs:
   vue:
     display_name: Vue
     build_command: npm run build
-    build_output_directory: dist
+    build_output_directory: /dist/browser/
```
Comment (petebacondarwin) [inline content/pages/_partials/_build-configuration.md]: This change doesn't look very Angular related?

Comment (petebacondarwin) [reply content/pages/_partials/_build-configuration.md]: Did you mean to chang the Angular build_output_directory?

File: content/pages/_partials/_build-configuration.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ build_configs:
     icon: /icons/framework-icons/logo-astro.svg
   angular-cli:
     display_name: Angular (Angular CLI)
-    build_command: npx ng build --prod
+    build_command: npx ng build --configuration production
     build_output_directory: dist
```
Comment (petebacondarwin) [inline content/pages/_partials/_build-configuration.md]: I think this should be

```suggestion
    build_output_directory: dist/cloudflare
```

**Review by petebacondarwin — CHANGES_REQUESTED** [review]

File: content/pages/_partials/_build-configuration.md (hunk 1 above)
Comment (markdembo) [reply content/pages/_partials/_build-configuration.md]: Wow. Let me fix - sorry

Comment (markdembo) [reply content/pages/_partials/_build-configuration.md]: Done. More coffee next time ;)

File: content/pages/_partials/_build-configuration.md (hunk 2 above)
Comment (markdembo) [reply content/pages/_partials/_build-configuration.md]: Done

**Review by petebacondarwin — APPROVED** [review]
LGTM - disclaimer: I haven't actually tested this in a Pages git connected project.

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**